resolver/type_syntax.cc

```diff
@@ -463,6 +463,22 @@ ParsedSig parseSigWithSelfTypeParams(core::Context ctx, const ast::Send &sigSend
     return sig;
 }
 
+std::optional<std::string> autoCorrectOrOfOnlyClasses(core::Context ctx, core::TypePtr type) {
+    if (auto *o = core::cast_type<core::OrType>(type) ) {
+        auto left = autoCorrectOrOfOnlyClasses(ctx, o->left);
+        auto right = autoCorrectOrOfOnlyClasses(ctx, o->right);
+        if (left && right) {
+            return fmt::format("T.any({}, {})", left.value(), right.value());
+        } else {
+            return {};
+        }
+    } else if (core::isa_type<core::ClassType>(type)) {
+        return fmt::format("T.class_of({})", type.show(ctx.state));
```

Comment:
```suggestion
        return fmt::format("T.class_of({})", type.show(ctx));
```

(ctx will be implicitly converted into a GlobalState. no need to type `ctx.state` manually)